tests/failpoints/cases/test_snap.rs

```diff
@@ -545,24 +545,33 @@ fn test_sending_fail_with_net_error() {
     let mut cluster = new_server_cluster(1, 2);
     configure_for_snapshot(&mut cluster);
 
-    let on_send_store_fp = "receiving_snapshot_net_error";
```

Comment:
why remove it？

Reply:
not move ,only use string  literal at line 566